### PR TITLE
Created a new hello world function - 

### PR DESCRIPTION
**Pull request by 254dev**
Branch created with Boniface

### CODE REVIEW

File: first_project.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+def hello_world():
+	print("Good afternoon")
+	return True
+
+if __name__ == '__main__':
+	hello_world()
```
Comment (bmwenda) [inline first_project.py]: Add a new line here

Comment (254dev) [reply first_project.py]: noted

Comment (bmwenda) [reply first_project.py]: Looks good now!

Comment (254dev) [reply first_project.py]: Good. Proceeding now

**Pull request merged**